docker-image/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:11.14.0-alpine
+
+RUN apk add \
+    ca-certificates \
+    musl-dev 
+
+ADD dist/ /graphql-voyager/dist/
+ADD middleware/ /graphql-voyager/middleware/
+ADD src/ /graphql-voyager/src/
+ADD package.json /graphql-voyager/package.json
+ADD docker-image/package.json /graphql-voyager/docker-image/package.json
+ADD docker-image/package-lock.json /graphql-voyager/docker-image/package-lock.json
+
+RUN npm install
+
+ADD docker-image/index.js /graphql-voyager/docker-image/index.js
+
+WORKDIR /graphql-voyager/
+
+WORKDIR /graphql-voyager/docker-image/
+
+RUN npm install
+
+ENV PORT=3001
+
+CMD node index.js
```

Comment:
missing newline at the end of the file.

Reply:
I actually like to have Dockerfile similar to what `graphql-faker` has:
https://github.com/APIs-guru/graphql-faker/blob/master/Dockerfile
Is it possible?

Reply:
Especially since it doesn't require a separate folder and `package.json`.

Reply:
> missing newline at the end of the file.

Fixed.

Reply:
> Especially since it doesn't require a separate folder and package.json.

I need a place to store `index.html` and this `package.json`:

```
{
  "scripts": {
    "start": "node index.js"
  },
  "dependencies": {
    "express": "^4.16.3",
    "graphql-voyager": "../"
  }
}
```

I don't want put the mess in root folder.

@IvanGoncharov  What do you think of that?